prototype/link/scripts/preprocessing_exploration/preprocessing_exploration.sh

```diff
@@ -89,7 +89,7 @@ echo Cycling through analyses
 # Take in the second input if there is one and use that to determine the analysis types
 if [ $# -eq 1 ]
 then
-analysis_types="default smoothing_0 smoothing_3 smoothing_5 smoothing_8 MELODIC_thresh_0.25_fslmotion_thr3 MELODIC_thresh_0.5_fslmotion_thr3 MELODIC_thresh_1.00_fslmotion_thr3 MotionParameters_Extended MotionParameters_Extended_confounds MotionConfounds_fslmotion_thr0.5 MotionConfounds_fslmotion_thr1 MotionConfounds_fslmotion_thr3 MotionConfounds_fslmotion_thr6 MotionConfounds_fslmotion_thr9 MotionConfounds_fslmotion_thr12 MotionConfounds_PCA_thr0_fslmotion_thr3 MotionConfounds_PCA_thr0.05_fslmotion_thr3 MotionConfounds_PCA_thrIQR_fslmotion_thr3 Despiking_None Temporal_derivative Motion_recovery_1 Motion_recovery_2 Regress_out_excluded"
+analysis_types="default smoothing_0 smoothing_3 smoothing_5 smoothing_8 MELODIC_thresh_0.25_fslmotion_thr3 MELODIC_thresh_0.5_fslmotion_thr3 MELODIC_thresh_1.00_fslmotion_thr3 MotionParameters_Extended MotionParameters_Extended_confounds MotionConfounds_fslmotion_thr0.5 MotionConfounds_fslmotion_thr1 MotionConfounds_fslmotion_thr3 MotionConfounds_fslmotion_thr6 MotionConfounds_fslmotion_thr9 MotionConfounds_fslmotion_thr12 MotionConfounds_PCA_thr0_fslmotion_thr3 MotionConfounds_PCA_thr0.05_fslmotion_thr3 MotionConfounds_PCA_thrIQR_fslmotion_thr3 Despiking_None Temporal_derivative Motion_recovery_1 Motion_recovery_2 Regress_out_excluded MotionConfounds_fslmotion_thr0.4 MotionConfounds_fslmotion_thr0.6 MotionConfounds_fslmotion_thr0.8 MotionConfounds_fslmotion_thr1.2 MotionConfounds_fslmotion_thr1.4 MotionConfounds_fslmotion_thr1.6 MotionConfounds_fslmotion_thr1.8 MotionConfounds_fslmotion_thr2 MotionConfounds_fslmotion_thr2.2 MotionConfounds_fslmotion_thr2.4 MotionConfounds_fslmotion_thr2.6 MotionConfounds_fslmotion_thr2.8 MotionConfounds_fslmotion_thr3.2 MotionConfounds_fslmotion_thr3.4 MotionConfounds_fslmotion_thr3.6 MotionConfounds_fslmotion_thr3.8 MotionConfounds_fslmotion_thr4 MotionConfounds_fslmotion_thr4.2 MotionConfounds_fslmotion_thr4.4 MotionConfounds_fslmotion_thr4.6 MotionConfounds_fslmotion_thr4.8 MotionConfounds_fslmotion_thr5 MotionConfounds_fslmotion_thr6 MotionConfounds_fslmotion_thr12 MotionConfounds_fslmotion_thr99 MotionConfounds_fslmotion_thr0.2 MotionParameters_Standard"
```

Comment:
May be annoying to do and not worth it but this is easier to read if they are in some type of order (all of the smoothing options are together, which is nice, but all of the motion thresholds are not because they were added at the end).

Reply:
Yeah it definitely isn't easy to read but the order matches the way the private branch works and would be inconvenient to change. I don't think it is a big concern for this line to be hard to read